### DISCUSSION TITLE
One channel disabled after reboot

### DISCUSSION THREAD

**Discussion by ckar69** [nextgenhealthcare/connect#4921] [Q&A]
Hello,
i have 2 groups with one channel each. After a reboot one channel got disabled, and the other one in the other group was up and running.
The two channels sources are TCP Listeners.

The mirth logs do not contain an exception of some sort. Did anybody faced anything similar? And what was a possible root cause?

Thank you in advance.

**Comment by rjkroll** [comment]
Do you have both channels set to Deployed/Enabled status so that they are both up and running after reboot?    If they are TCP listeners, make sure you didnt accidently use the same listening port.   Other then that, when you try to enable to the problem channel, there should be something in the logs if it doesn't deploy/enable.   Channels just do not deploye/enable for no reason unless you set them not to.

**Comment by ckar69** [comment]
Hi rjkroll,
well no, the channels are TCP Listeners, but do not share the same port. I looked at the logs, but nothing seemed to be wrong. I compared the two TCP listeners, and the only diferrence was that the disabled_after_reboot channel had "Keep Connection Open" off. All other setting were the same.
I tried to reproduce, by keeping this setting off, but in the next on_purpose reboot, the channel was enabled.
Maybe something in the network level (Keep Alive?) turned the channel off, due to inactivity maybe?

**Reply by pacmano1** [reply]
Never seen this happen.  Unless someone disabled the channel in the channel tab and someone forgot about it.   When the server rebooted and auto-deployed channels, the non-enabled one would not deploy obviously.   Obviously some other source or automation someone forgot about could do this as well.
